easyconfigs/s/spaCy/spaCy-3.4.4-foss-2022a.eb

```diff
@@ -0,0 +1,92 @@
+easyblock = 'PythonBundle'
+
+name = 'spaCy'
+version = '3.4.4'
+
+homepage = 'https://spacy.io/'
+description = "Industrial-strength Natural Language Processing (NLP) in Python."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+use_pip = True
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+]
+
+exts_list = [
+    ('spacy-legacy', '3.0.10', {
+        'checksums': ['16104595d8ab1b7267f817a449ad1f986eb1f2a2edf1050748f08739a479679a'],
+    }),
+    ('spacy-loggers', '1.0.4', {
+        'checksums': ['e6f983bf71230091d5bb7b11bf64bd54415eca839108d5f83d9155d0ba93bf28'],
+    }),
+    ('cymem', '2.0.7', {
+        'checksums': ['e6034badb5dd4e10344211c81f16505a55553a7164adc314c75bd80cf07e57a8'],
+    }),
+    ('murmurhash', '1.0.9', {
+        'checksums': ['fe7a38cb0d3d87c14ec9dddc4932ffe2dbc77d75469ab80fd5014689b0e07b58'],
+    }),
+    ('preshed', '3.0.8', {
+        'checksums': ['6c74c70078809bfddda17be96483c41d06d717934b07cab7921011d81758b357'],
+    }),
+    ('blis', '0.7.9', {
+        'checksums': ['29ef4c25007785a90ffc2f0ab3d3bd3b75cd2d7856a9a482b7d0dac8d511a09d'],
+    }),
+    ('confection', '0.0.3', {
+        'checksums': ['4fec47190057c43c9acbecb8b1b87a9bf31c469caa0d6888a5b9384432fdba5a'],
+    }),
+    ('thinc', '8.1.6', {
+        'checksums': ['9241c37761f004fe684e637d2b4d8b79addebabc64e343aa1cba144fad2c9b47'],
+    }),
+    ('ml_datasets', '0.2.0', {
+        'checksums': ['3f9c8901f8d6be3dab5b23ec3a6c01e619a60d0184696b1030cde2e3086943f1'],
+    }),
+    ('wasabi', '0.10.1', {
+        'checksums': ['c8e372781be19272942382b14d99314d175518d7822057cb7a97010c4259d249'],
+    }),
+    ('srsly', '2.4.5', {
+        'checksums': ['c842258967baa527cea9367986e42b8143a1a890e7d4a18d25a36edc3c7a33c7'],
+    }),
+    ('catalogue', '2.0.8', {
+        'checksums': ['b325c77659208bfb6af1b0d93b1a1aa4112e1bb29a4c5ced816758a722f0e388'],
+    }),
+    ('typer', '0.7.0', {
+        'checksums': ['ff797846578a9f2a201b53442aedeb543319466870fbe1c701eab66dd7681165'],
+    }),
+    ('pathy', '0.10.1', {
+        'checksums': ['4cd6e71b4cd5ff875cfbb949ad9fa5519d8d1dbe69d5fc1d1b23aa3cb049618b'],
+    }),
+    ('smart-open', '6.3.0', {
+        'source_tmpl': 'smart_open-%(version)s.tar.gz',
+        'checksums': ['d5238825fe9a9340645fac3d75b287c08fbb99fb2b422477de781c9f5f09e019'],
+    }),
+    ('tqdm', '4.64.1', {
+        'checksums': ['5f4f682a004951c1b450bc753c710e9280c5746ce6ffedee253ddbcbf54cf1e4'],
+    }),
```

Comment:
Can we use the existing https://bear-apps.bham.ac.uk/applications/2022a/tqdm/4.64.0-GCCcore-11.3.0/ for this?